specification/Metadata/0.0.0/README.md

```diff
@@ -439,7 +439,7 @@ The JSON encoding is the simplest encoding, designed to be human-readable.
 
 #### Basic Types
 
-For JSON encoding, basic types include integers, floating point numbers, booleans, and strings. These are encoded in the most natural type available in JSON. Numeric types are represented as `number`, booleans as `boolean`, and `string` as `string`.
+For JSON encoding, basic types include integers, floating point numbers, booleans, and strings. These are encoded in whatever JSON type is most. Numeric types are represented as `number`, booleans as `boolean`, and strings as `string`.
```

Comment:
```suggestion
For JSON encoding, basic types include integers, floating point numbers, booleans, and strings. These are encoded in whatever JSON type is most natural. Numeric types are represented as `number`, booleans as `boolean`, and strings as `string`.
```

Reply:
Also, is natural the right word here? Maybe "appropriate" is better?

Reply:
hm yeah, "appropriate" sounds better here.